### PR TITLE
Fix upload for binary files.

### PR DESCRIPTION
**Pull request by mdwelsh**

### CODE REVIEW

File: packages/fixie/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fixie",
-  "version": "1.3.1",
+  "version": "1.3.2",
```
Comment (NickHeiner) [inline packages/fixie/package.json:3]: Did you publish this on the PR? 

```
npm info fixie@1.3.2
npm WARN Ignoring workspaces for specified package(s)

fixie@1.3.2 | MIT | deps: 12 | versions: 8
This package contains an SDK and command-line interface to the [Fixie.ai](https://fixie.ai) platform.
https://fixie.ai

bin: fixie

dist
.tarball: https://registry.npmjs.org/fixie/-/fixie-1.3.2.tgz
.shasum: d2bcc537b0e88d8327f351d4e6256609ea61d853
.integrity: sha512-mZmw7kvFiUVBOuJEQDA8RGvikslfopDFY2tTfBAu4UWf7h4xEeegNLdZLu+9DD+8x7N9FaSQi3zAoUF80f8xMQ==
.unpackedSize: 61.1 kB

dependencies:
@apollo/client: ^3.8.1        axios: ^1.5.0                 execa: ^8.0.1                 graphql: ^16.8.0              open: ^9.1.0                  terminal-kit: ^3.0.0
apollo-upload-client: ^17.0.0 commander: ^11.0.0            extract-files: ^13.0.0        js-yaml: ^4.1.0               ora: ^7.0.1                   untildify: ^5.0.0

maintainers:
- nick.heiner <nth23@cornell.edu>
- benlower <npm@benlower.us>
- juberti <justin@uberti.name>
- mdwelsh <mdw@mdw.la>

dist-tags:
latest: 1.3.2

published 22 hours ago by mdwelsh <mdw@mdw.la>
```

Publishing off an unlanded PR is generally not a good idea. If you wanted to publish a pre-release version, do `1.3.2-0`: https://semver.org/#spec-item-9.

This caused a confusing situation with my PR, where I also landed `1.3.2`, then published it, then saw that I was unable to do so.

Comment (mdwelsh) [reply packages/fixie/package.json:3]: Uhhh ... I didn't *mean* to, but it would not have been beyond my abilities

**Review by hessamb — APPROVED** [review]
This PR LGTM btw!

File: packages/fixie/package.json (hunk 1 above)
Comment (hessamb) [reply packages/fixie/package.json:3]: I think you published to test. Maybe upgrade directly to 1.3.3 in this PR. And after it's merged @NickHeiner can publish to include his changes.

**Pull request merged**